lib/options.ts

```diff
@@ -15,6 +15,7 @@ export class Options extends commonOptionsLibPath.OptionsBase {
 			forDevice: { type: OptionType.Boolean },
 			provision: { type: OptionType.Object },
 			client: { type: OptionType.Boolean, default: true },
+			env: { type: OptionType.Object },
```

Comment:
what about uglify?

Reply:
uglify will be passed as a regular options - `--env.uglify`, we've agreed with @sis0k0 on that behavior

Reply:
The same applies for `--env.snapshot`